src/bullets/data_source/recorded_data.py

```diff
@@ -1,3 +1,5 @@
+from datetime import datetime
+
```

Comment:
Sorry for nitpicking, but there shouldn't be a blank line here.

Reply:
No prob, fixed :)